testing/bird/APKBUILD

```diff
@@ -32,49 +32,39 @@ _build() {
 		--sysconfdir=/etc \
 		--mandir=/usr/share/man \
 		--localstatedir=/var \
-		$@ \
-		|| return 1
-	make || return 1
+		$@
```

Comment:
Could you change it to `"$@"` here? According to POSIX, `"$@"` is the only way to reproduce the received argument line with proper quoting itself. It's a bit of a special case.